OpenRA.Mods.Common/Widgets/ScrollPanelWidget.cs

```diff
@@ -177,6 +177,9 @@ void Scroll(int amount, bool smooth = false)
 			newTarget = Math.Min(0, Math.Max(Bounds.Height - ContentHeight, newTarget));
 
 			SetListOffset(newTarget, smooth);
+
+			foreach (var child in Children)
+				child.MouseExited();
```

Comment:
If it was only a small scroll then chances are that it hasn't exited the children.